tests/flows/jobs/test_process_aws_invitations.py

```
@@ -0,0 +1,134 @@
+        return_value=mock_aws_client,
+    )
+    notification_mock = mocker.patch(
+        "swo_aws_extension.flows.jobs.process_aws_invitations.TeamsNotificationManager"
+    )
+
+    processor.process_aws_invitations()  # act
+
+    notification_mock.return_value.notify_one_time_error.assert_called_once()
+
+
+def test_process_invitations_no_orders(mocker, config, mpt_client, requests_mocker):
```

Ruff (0.14.10)

Unused function argument: `mocker`

(ARG001)